packages/headless/src/insight.index.ts

```diff
@@ -92,6 +85,14 @@ export type {
   CategoryFacetSearch,
   CategoryFacetSearchState,
   CategoryFacetSearchResult,
+  /**
+   * @deprecated This is an internal controller that will be removed in the next version.
+   */
+  CoreCategoryFacet,
+  /**
+   * @deprecated This is an internal controller that will be removed in the next version.
```

Comment:
Could it be that we need to keep exporting these core interfaces for documentation purposes?

Reply:
I think that's it. I added them to fix errors the doc parser was throwing.